test/src/ir_generation/mod.rs

```diff
@@ -141,7 +141,7 @@ pub(super) async fn run(filter_regex: Option<&regex::Regex>) -> Result<()> {
                 let tree_type = typed_program.kind.tree_type();
 
                 // Compile to IR.
-                let include_tests = false;
+                let include_tests = true;
```

Comment:
To have a test to check metadata generated for the tests I need it to be compiled with `include_tests` enabled. I can change this to to be true if the file name starts with `test_` or something similar to not the change default behavior for all IR tests if it is preferred over this. cc @otrho

Reply:
I don't see why we shouldn't try to have `include_tests` as true.  @mitchmindtree was probably being conservative, unless it overly complicates things somehow?  Most of the existing tests won't have any unit tests so it wouldn't make any difference.